### PR TITLE
Updating uhttp package

### PR DESCRIPTION
**Pull request by mchavez**
- Updating uhttp package

### CODE REVIEW

File: pkg/uhttp/wrapper.go [hunk 1]
```diff
@@ -215,6 +232,30 @@ func WithAcceptXMLHeader() RequestOption {
 	return WithHeader("Accept", "application/xml")
 }
 
+func WithContentTypeFormHeader() RequestOption {
```
Comment (mgaeta) [inline pkg/uhttp/wrapper.go:235]: It looks like there are already some helper functions so you could do
```go
func WithContentTypeFormHeader() RequestOption {
	return WithHeader("Content-Type", "application/x-www-form-urlencoded")
}
...
func WithAcceptVndJSONHeader() RequestOption {
	return WithHeader("Accept", "application/vnd.api+json")
}
```
or add new helpers and do something like:
```go
func WithContentType(value string) RequestOption {
	return WithHeader("Content-Type", value)
}
func WithContentTypeFormHeader() RequestOption {
	return WithContentType("application/x-www-form-urlencoded")
}
...
```

Comment (mchavez) [reply pkg/uhttp/wrapper.go:235]: Yes. Refactoring is always valid. Thanks..!!

Comment (mchavez) [reply pkg/uhttp/wrapper.go:235]: Fixed

**Comment by ggreer** [comment]
Closing because https://github.com/ConductorOne/baton-sdk/pull/193 addresses this.

**Pull request closed without merging**